src/main/java/Parser.java

```diff
@@ -0,0 +1,124 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+
+
+    /**
+     * Parse the input string into recognisable command.
+     *
+     * @param string
+     * @return
+     * @throws DukeException
+     */
+    public static Command parse(String string) throws DukeException {
+        String action, info;
+        Command command;
+
+        if (string.contains(" ")){
```

Comment:
There should be a space between the ) and the {

Reply:
Thanks for the input.